src/web/controller.py

```diff
@@ -37,11 +40,15 @@ def get(self, scenario_id: str) -> Scenario2D:
             The built-in scenario.
         """
         if scenario_id not in self.scenarios:
+            self.scenarios[scenario_id] = {}
+
+        if center_frequency not in self.scenarios[scenario_id]:
             builtin_scenario = BUILT_IN_SCENARIOS[scenario_id]()
+            builtin_scenario.center_frequency = center_frequency
             builtin_scenario.make_grid()
             builtin_scenario.compile_problem()
-            self.scenarios[scenario_id] = builtin_scenario
-        return self.scenarios[scenario_id]
+            self.scenarios[scenario_id][center_frequency] = builtin_scenario
+        return self.scenarios[scenario_id][center_frequency]
```

Comment:
Do you know why most `center_frequency` values are fine, but a few (e.g. `500005`) error out?

https://www.loom.com/share/d10193b3d2434a7aab180db14d2d9b35

Reply:
oh, thanks for pointing this out! 
It's an assertion added to the compute grid shape function: https://github.com/agencyenterprise/neurotechdevkit/blob/main/src/neurotechdevkit/grid.py#L23
So it would also fail when executed outside of the web UI.

I don't know why is it required, and I don't know if we should handle this in the web UI. 

@d-lucena what do you think?

Reply:
Oh, I guess the assertion's idea is that the `grid` should exactly match the specified `extent` size. I'll leave it to @d-lucena to comment on whether we still want that. If so, perhaps the web UI could instead be a drop-down of evenly-dividing center-frequencies.